test/mithril.trust.js

```diff
@@ -64,5 +64,18 @@ describe("m.trust()", function () {
 			expect(root.innerHTML).to.equal("<div><p>&amp;copy;</p><p>©</p>©</div>")
 		})
 
+		// not sure that this goes here; not really an m.trust issue
```

Comment:
I think it belongs in `test/mithril.render.js`, not here. You comment can be removed, and I feel it's safe to merge, as long as tests are passing.